### ISSUE TITLE
Try using es6 generators

### ISSUE THREAD

**Issue by Bikossor** [Bikossor/anonymus#54]

**Comment by StringKe** [comment]
esm import error.
```
import anonymus from "anonymus";
console.log(anonymus) // undefined
```

**Comment by Bikossor** [comment]
@StringKe this does not really belong to this issue but try the following:
```ts
import * as anonymus from "anonymus";

console.log(anonymus);
```

**Comment by StringKe** [comment]
> 

it worked, thanks

**Comment by Bikossor** [comment]
This will be implemented with #306
